vendor/processors/stripe_processor.py

```diff
@@ -558,29 +575,32 @@ def get_coupons(self):
         return coupons_list
 
     def get_vendor_offers_in_stripe(self, offer_pk_list, site):
-        offers = Offer.objects.filter(site=site, pk__in=offer_pk_list)
+        offers = Offer.objects.filter(site=site, pk__in=offer_pk_list, meta__has_key='stripe')
```

Comment:
Not sure if this is really helping. Again what happens if the offer is in that list and does not have the stripe key. When would that record be updated to have it?

Reply:
Isnt our assumption that anything that needs to be updated, should already have a stripe key in its meta? If it doesn't have that key, then its assumed it was never created in stripe and will be created in the next steps

Reply:
Glad we could figure out the edge cases togther. Nice work